modules/auxiliary/server/capture/http.rb

```diff
@@ -3,6 +3,8 @@
 # Current source: https://github.com/rapid7/metasploit-framework
 ##
 
+require 'rex/proto/http/server'
```

Comment:
Not tested: I believe if you extract `ServerClient` into its own `lib/rex/proto/http/server_client.rb` file this should autoload without the need for `require` here

Reply:
Happy to do that if you think it's the better solution.

Reply:
I think so, it'll stop anything like this happening again in the future - and as it looks like it's part of the rex proto public API it seems like a fair thing to do

Hopefully it works 🤞

Reply:
That fixed the issue as well, so I reverted the original change. Thanks for the suggestion.